### DISCUSSION TITLE
The repository lpigeon has been retired and cannot be reused

### DISCUSSION THREAD

**Discussion by lpigeon** [community/community#44875] [New to GitHub]
I recently changed my username from sabi-github to lpigeon (current username is sabi-github) to reuse username(lpigeon).  And I founded the error massage 'The repository lpigeon has been retired and cannot be reused.' and 'The repository TIL has been retired and cannot be reused.'. How I can fix this error? 

Regards,
Jungsoo Lee

**Comment by queenofcorgis** [comment]
#23480 - check out this Discussion to see if this helps!

**Reply by lpigeon** [reply]
thanks! it helped me a lot!
